notice_and_comment/static/regulations/css/less/layout-custom.less

```diff
@@ -0,0 +1,14 @@
+/*
+ Layout
+ ==========================================================================
+ layout.less defines the structural layout of the site
+ */
+
+.panel {
+    position: fixed;
+    top: 90px;
+    bottom: 0px;
```

Comment:
No need to use a unit with 0.
https://pages.18f.gov/frontend/css-coding-styleguide/units/